modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -22,18 +22,20 @@
         :is-modified="isModified"
         :is-modified-from-published="isModifiedFromPublished"
         :can-discard-draft="canDiscardDraft"
+        :can-delete="canDelete"
         :can-archive="canArchive"
-        :can-copy="!!docId && !moduleOptions.singleton"
+        :can-copy="!!docId && !moduleOptions.singleton && !archived"
```

Comment:
The "More" menu now appears in more circumstances because "Delete" is available in the doc editor even when archived, so I added more guards against offering other options at inappropriate times.